client/src/views/editorModal.tsx

```diff
@@ -1,7 +1,7 @@
 import './editorModal.scss';
 import React from 'react';
 import Modal from 'react-modal';
-import yamljs from 'yamljs';
+import Yaml from 'yaml';
```

Comment:
Supper small nitpick, but would you mind changing `Yaml` to `yaml` (i.e. lowercase)? An uppercase indicates it's a React component. Thx!

Reply:
@herbrandson Done. 
Got to learn more about `React component`. Thank you!